blockmanager.go

```diff
@@ -98,12 +99,39 @@ type txMsg struct {
 	peer *ServerPeer
 }
 
+// blockManagerCfg holds options and dependencies needed by the blockManager
+// during operation.
+type blockManagerCfg struct {
+	// ChainParams is the chain that we're running on.
+	ChainParams chaincfg.Params
+
+	BlockHeaders     headerfs.BlockHeaderStore
+	RegFilterHeaders *headerfs.FilterHeaderStore
+	TimeSource       blockchain.MedianTimeSource
+
+	BanPeer func(addr string, reason banman.Reason) error
+
+	// GetBlock fetches a block from the p2p network.
+	GetBlock func(chainhash.Hash, ...QueryOption) (*btcutil.Block, error)
+
+	queryBatch func([]wire.Message, func(*ServerPeer, wire.Message,
```

Comment:
IMO either all the values should be private, or all public.

Reply:
These closures will be removed.